nextcord/file.py

```diff
@@ -78,15 +84,31 @@ class File:
         in the Discord client.
     spoiler: :class:`bool`
         Whether the attachment is a spoiler.
+    force_close: :class:`bool`
+        Whether to forcibly close the bytes used to create the file
+        when ``.close()`` is called.
+        This will also make the file bytes unusable by flusing it from
+        memory after it is sent or used once.
+        Keep this enabled if you don't wish to reuse the same bytes.
```

Comment:
```suggestion
        Whether to forcibly close the bytes used to create the file
        when ``.close()`` is called.
        This will also make the file bytes unusable by flushing it from
        memory after it is sent or used once.
        Enable this if you don't wish to reuse the same bytes.
```